website/source/guides/identity/secure-intro.html.md

```diff
@@ -0,0 +1,102 @@
+  This introductory guide walk through the mechanism of Vault clients to
+  authenticate with Vault. There are two approaches at a high-level: platform
+  integration, and trusted orchestrator.
+---
+
+# Secure Introduction of Vault Clients
+
+A _secret_ is something that will elevate the risk if exposed to unauthorized
+entities and results in undesired consequences (e.g. unauthorized data access);
+therefore, only the ***trusted entities*** should have an access to your
+secrets.
+
+If you can securely get the first secret from an originator to a consumer,
+all subsequent secrets transmitted between this originator and consumer can be
+authenticated with the trust established by the successful distribution and user
+of that first secret.  
+
+![Secure Introduction](/assets/images/vault-secure-intro-1.png)
+
+The Vault authentication process verifies the secret consumer's identity and
+then generate a **token** to associate with that identity.
+[Tokens](/docs/concepts/tokens.html) are the core method for authentication
+within Vault which means that the secret consumer must first acquire a valid
+token.
+
+
+## Challenge
+
+How does a secret consumer (an application or machine) prove that it is the
+legitimate recipient for a secret so that it can acquire a token?
+
+How can you avoid persisting raw token values during our secure
+introduction?  
+
+## Secure Introduction Approach
+
+Vault's auth methods perform authentication of its client and assigning a set of
+policies which defines the permitted operations for the client.
+
+![Auth Method](/assets/images/vault-auth-method.png)
+
+There are two basic patterns to securely authenticate a secret consumer:
+[platform integration](#platform-integration) and [trusted
+orchestrator](#trusted-orchestrator).  
+
+
+### Platform Integration
+
+In the **Platform Integration** model, Vault trusts the underlying platform
+(e.g. AWS, Azure, GCP) which assigns an identifier to its cloud resources (e.g.
+an IAM token, instance ID, JWT). The Vault client (secret consumer)
+authenticates with Vault using its platform provided identifier. Once its
+identity was successfully validated against the platform, Vault returns an
+initial token to the client with a set of configured policies attached.
+
+![Platform Integration](/assets/images/vault-secure-intro-2.png)
+
+**Use Case**
+
+When the client app is running on a VM hosted on a supported cloud platform, you
+can leverage the corresponding auth method to authenticate with Vault.
+
+**Reference Materials:**
+
+- [AWS Auth Method](/docs/auth/aws.html)
+- [Azure Auth Method](/docs/auth/azure.html)
+- [GCP Auth Method](/docs/auth/azure.html)
+
+### Trusted Orchestrator
+
+In the **Trusted Orchestrator** model, you have an _orchestrator_ which is
+already authenticated against Vault with privileged permissions. The
+orchestrator launches new applications and inject a mechanism they can use to
+authenticate (e.g. AppRole, PKI cert, token, etc) with Vault.
+
```

Comment:
I would add an example here as well:

"For example, suppose Terraform is being used as a trusted orchestrator. This means Terraform already has a Vault token, with enough capabilities to generate new tokens or create new mechanisms to authenticate such as an AppRole. Terraform can interact with platforms such as VMware to provision new virtual machines. VMware does not provide a cryptographic identity, so a platform integration isn't possible. Instead, Terraform can provision a new AppRole credential, and SSH into the new machine to inject the credentials. Terraform is creating the new credential in Vault, and making that credential available to the new resource. In this way, Terraform is acting as a trusted orchestrator and extending trust to the new machine. The new machine, or application running on it, can use the injected credentials to authenticate against Vault.